eva_cttv_pipeline/efo_term.py

```diff
@@ -66,37 +74,25 @@ def __ne__(self, other):
     def __repr__(self):
         return "%s(%r)" % (self.__class__, self.__dict__)
 
-    @property
-    def efoid(self):
-        return self.__efoid
-
-    @efoid.setter
-    def efoid(self, value):
-        self.__efoid = value
-
     def is_obsolete(self):
         if self.efoid is not None and self.efoid in EFOTerm.obsolete_terms:
             raise EFOTerm.IsObsoleteException(
-                "Term " + self.efoid + " is obsolete. Cause/Description/Action: " + EFOTerm.obsolete_terms[self.efoid])
+                "Term " + self.efoid + " is obsolete. Cause/Description/Action: " +
+                EFOTerm.obsolete_terms[self.efoid])
         else:
             return False
 
     def is_cttv_available(self):
         if self.efoid is not None and self.efoid in EFOTerm.cttv_available_terms:
             return True
         else:
-            raise EFOTerm.NotCttvAvailableException("Term " + self.efoid + " is not currently available at EFO.")
+            raise EFOTerm.NotCttvAvailableException("Term " + self.efoid +
+                                                    " is not currently available at EFO.")
```

Comment:
This exception class name could be misleading, it doesn't represent a lack of CTTV service availability but a term's.